examples/MomentoLoadGen/Program.cs

```diff
@@ -407,45 +393,45 @@ static ILoggerFactory InitializeLogging()
 
         static async Task Main(string[] args)
         {
-            using (ILoggerFactory loggerFactory = InitializeLogging())
+            CsharpLoadGeneratorOptions loadGeneratorOptions = new CsharpLoadGeneratorOptions(
+              ///
+              /// Controls the verbosity of the output during the run.
+              ///
+              logLevel: LogLevel.Debug,
+              ///
+              /// Each time the load generator has executed this many requests, it will
+              /// print out some statistics about throughput and latency.
+              ///
+              showStatsInterval: new TimeSpan(0, 0, 5),
```

Comment:
one small nit - I believe the TimeSpan class has some static methods that are factory constructors that are something like "ofSeconds" or "ofMinutes".  I find those to be easier to read/review.

Reply:
And I forgot to change the comment. 🙄